pkg/parser/sysl.wbnf

```diff
@@ -85,7 +85,8 @@ PATH    -> /{[a-zA-Z0-9._-]+}:"/";
 DIGITS -> [1-9][0-9]*;
 TEXT -> [^(\r?\n)]+;
 TEXT_LINE -> /{\s* ( [^\s:(] [^(\r\n:]* [^\s\r\n(:] )};
-APPLICATION_NAME -> (pkg=@):"." > (app=@):"::" > (NAME | QSTRING)+;
+APPLICATION_NAME -> (pkg=@):"." > (app=@):"::" > APPLICATION_NAME_PART;
+APPLICATION_NAME_PART -> (/{[a-zA-Z_][-\w]*(?:[-\t\_]*\w+)*} | QSTRING)+;
```

Comment:
I'd really much prefer `APPNAME`. It's pretty unambiguous and much more compact.